.github/workflows/ci-github-actions.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v3
       - id: go-version
         # Reference: https://github.com/actions/setup-go/issues/23
-        run: echo "::set-output name=version::$(cat ./.go-version)"
+        run: echo "version=$(cat ./.go-version)" >> "$GITHUB_OUTPUT"
```

Comment:
I'm not convinced this is correct.

It was the only way I found to silence the following error:
```shell
shellcheck reported issue in this script: SC2086:info:1:19: Double quote to prevent globbing and word splitting [shellcheck]
```

But I can't see any other examples where `$GITHUB` needs to be quoted.

Reply:
If we'd like to keep the explicit `.go-version` file, the `actions/setup-go` action accepts a `go-version-file` input, which can be pointed at either a `.go-version`-like file, or `go.mod`. We opted to drop `.go-version` in other `terraform-plugin-*` projects though as latest patch version based on `go.mod` is fine for our usage and easier for maintenance.